packages/bitcore-wallet-service/src/lib/model/txproposal.ts

```diff
@@ -255,6 +259,8 @@ export class TxProposal {
     x.gasLimit = obj.gasLimit; // Backward compatibility for BWC <= 8.9.0
     x.data = obj.data; // Backward compatibility for BWC <= 8.9.0
     x.tokenAddress = obj.tokenAddress;
+    x.multisigContractAddress = obj.multisigContractAddress;
+    x.multisigTxId = obj.multisigTxId;
```

Comment:
this is the TXID that created the contract? Why is needed?

Reply:
That's the transaction id on the contract (  0, 1, 2, 3, etc ). Different from the transaction hash. Two transactions with different transactions hash have the same transaction id when they are trying to execute the contract. If I have a transaction that still need confirmations and I want to confirm one of them, I need the id to implement the confirm method.
```
  confirmTransactionEncodeData(params: { multisigContractAddress: string; transactionId: number }) {
    const { multisigContractAddress, transactionId } = params;
    const data = this.getMultisigContract(multisigContractAddress)
      .methods.confirmTransaction(transactionId)
      .encodeABI();
    return data;
  }
```